DESCRIPTION.md

```diff
@@ -9,6 +9,10 @@ Source code is also available at:
 
 # Release Notes
 
+- v1.4.5(Dec 7, 2022)
+
+  - Added telemetry for sqlalchemy usage information.
```

Comment:
Could please rewrite this? I know the application name is used for telemetry too, but really you updated the application name given to the connector

Reply:
you're correct, I've rewritten this.